scripts/lib/find_fastest_server.pl

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/perl -w
+use strict;
+my %best;
+foreach my $server (@ARGV) {
+    $_ = `ping -W 1 -c 3 -i .2 -q $server |tail -1`;
+    next unless (m/ = ([0-9.]+)\//);
+    my $time = $1;
+    if(!defined $best{"time"} || $time < $best{"time"}) {
+        %best = ("time"=>$time, "server"=>$server);
+    }
+    #print STDERR "$server $time\n"; # debug
+}
+$best{server} ||= "clouddata.nue.suse.com.";
```

Comment:
Maybe keep out the default. So the script is very agnostic about the servers tested.
Just take the last (or the first) one as default.